testplans/graphsync/memory-stress-k8s.toml

```diff
@@ -0,0 +1,40 @@
+[metadata]
+name = "memory-stress"
+
+[global]
+plan = "graphsync"
+case = "stress"
+total_instances = 2
+builder = "docker:go"
+runner = "cluster:k8s"
+
+[global.build_config]
+push_registry=true
+go_proxy_mode="remote"
+go_proxy_url="http://localhost:8081"
+registry_type="aws"
+
+[global.run.test_params]
+size      = "512MB"
+latencies = '["50ms"]'
+bandwidths = '["32MiB"]'
+concurrency = "1"
+chunk_size = "18"
+links_per_level = "1024"
+raw_leaves = "false"
+disk_store = "true"
+memory_snapshots = "detailed"
+
+[[groups]]
+id = "providers"
+instances = { count = 1 }
+[groups.resources]
+memory = "4096Mi"
```

Comment:
Running this test with the following params, results in OOMKilled error for the `provider`.

```
[global.run.test_params]
size      = "8GB"
latencies = '["20ms"]'
bandwidths = '["128MiB"]'


[[groups]]
id = "providers"
instances = { count = 1 }
[groups.resources]
memory = "2048Mi"
cpu = "1000m"

[[groups]]
id = "requestors"
instances = { count = 1 }
[groups.resources]
memory = "2048Mi"
cpu = "1000m"
```

Reply:
should be fixed now.

Reply:
I am still getting OOMKilled when I try to run the `memory-stress-k8s.toml` with `size` > `memory` for providers and requestors.